site/operational-guide/concepts.md

```diff
@@ -49,12 +49,16 @@ When the router is launched on a host, it derives its peer name in order of pref
 
 * From the command line, where the user is responsible for uniqueness and
   stability
-* From the BIOS product UUID, which is generally stable across
-  restarts and unique across different physical hardware and certain
-  cloned VMs
-* From the hypervisor UUID, which is generally stable across restarts
-  and unique across VMs which do not provide access to a BIOS product
-  UUID
+* From a product of concatenation of:
+    * The unique machine ID, which is generated either by [systemd](https://www.freedesktop.org/software/systemd/man/machine-id.html)
```

Comment:
The first sentence, edit to (if this is what was meant? any of these concatenated?). 

From the concatenated result of any of the following: